src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double firstLeg, double secondLeg, double height) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstLeg + secondLeg) * height) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: isosceles trapezoid, area: %.2f sq. cm, firstLeg: %.2f cm, "
```

Comment:
Avoid using `System.out.printf` directly in the `draw` method. Consider using an interface to abstract the drawing behavior, which could then be implemented by a class responsible for handling the output.